app/src/main/java/io/github/androidacademyglobal/ui/theme/ConstantType.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.androidacademyglobal.ui.theme
+
+const val FONT_SIZE_REGULAR = 16
+const val FONT_SIZE_SMALL = 12
```

Comment:
use same syntax for all const files:
snakeCase or CAMEL_CASE

Reply:
I'm using CAMEL_CASE for constants everywhere. Is that okay?

Reply:
please use same approach in all files.
As example
[ui/theme/Color.kt](https://github.com/aa-open-source-team/android-academy-app/pull/3/files#diff-3171b7a77a23f409b4bd0c6133b159c6702d3c87c3871b2e0637d718351343b9)

and please add namespace:

AppDimensions{
    const val FONT_SIZE_SMALL = 12
    ...
}

Reply:
added namespaces and annotations